src/main/java/com/rapid7/net/AsyncLogger.java

```
@@ -555,12 +553,13 @@ void reopenConnection() throws InterruptedException {
                 }
```

Is simply catching `IOException` sufficient above? `openConnection()` previously swallowed all exceptions.

---

should be sufficient.